backend/resume_evaluations/schemas.py

```
@@ -0,0 +1,39 @@
+    formatting_feedback: str
+    additional_value_feedback: str
+
+
+class ResumeEvaluationRequest(BaseModel):
+    """Request payload for resume evaluation."""
+
+    resume_text: str
+    job_description: str
+
+
+class ResumeEvaluationResponse(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)